src/state/PpeReducer.js

```diff
@@ -0,0 +1,10 @@
+export default (
```

Comment:
Could we update the reducers / state / actions to follow the 'redux ducks' approach. I've went into a bit more detail on how that works in a comment on the other repo, you can [read it here](https://github.com/AddJam/medicbleep-handwashtimer/pull/8#discussion_r417927126) - I hope that makes sense. Let me know if it doesn't.